client/app/scripts/superdesk-authoring/views/authoring-topbar.html

```diff
@@ -9,7 +9,7 @@
     </div>
 
     <div class="button-stack right-stack">
-        <button class="btn btn-medium pull-left" ng-if="item._locked && can_unlock() && itemActions.save" ng-click="unlock()" ng-disabled="unlockClicked" translate>Unlock</button>
+        <button class="btn btn-medium pull-left" ng-if="item._locked && !item.more_coming && can_unlock() && itemActions.save" ng-click="unlock()" ng-disabled="unlockClicked" translate>Unlock</button>
```

Comment:
This value is only set on send to, so only there is true and in all other cases its false, so that's why I added that.